lib/alephant/sequencer/sequence_table.rb

```diff
@@ -12,8 +12,10 @@ class SequenceTable < ::Alephant::Support::DynamoDB::Table
       attr_reader :table_name, :client
 
       def initialize(table_name)
+        options = {}
+        options.merge!({endpoint: ENV['AWS_DYNAMO_DB_ENDPOINT']}) if ENV['AWS_DYNAMO_DB_ENDPOINT']
```

Comment:
I made the same comment on the storage gem but the merge doesn't read very well for me. 

Is there a specific reason for the `merge`?
`options = ENV['AWS_S3_ENDPOINT'] ? {endpoint: ENV['AWS_S3_ENDPOINT']} : {}` perhaps?
or if you still want it separated out
```
options = {}
options[:endpoint] = ENV['AWS_S3_ENDPOINT'] if ENV['AWS_S3_ENDPOINT']
```